src/utils/tokens.ts

```diff
@@ -462,6 +539,28 @@ const tokens = {
       });
     }
 
+    if (data !== null) {
+      for (const dataString of data) {
+        const outputData = {
+          type: 'data',
+          data: dataString,
+          value: 1,
+          token: HATHOR_TOKEN_CONFIG.uid,
+          authorities: 0,
+        } as IDataOutput;
+
+        // We currently have an external service that identifies NFT tokens with the first output as the data output
+        // that's why we are keeping like this
+        // However, this will change after a new project is completed to better identify an NFT token
+        // the method that validates the NFT is in src/models/CreateTokenTransaction.validateNft
+        if (unshiftData) {
+          outputs.unshift(outputData);
+        } else {
+          outputs.push(outputData);
+        }
+      }
+    }
```

Comment:
Can this be a helper?

Reply:
I like the DRY (Don't Repeat Yourself) thinking but since this logic is only used twice in the lib I don't mind leaving it repeated.